pystiche_papers/li_wand_2016/_nst.py

```diff
@@ -26,6 +26,30 @@ def nst(
         Callable[[int, Union[torch.Tensor, pystiche.LossDict]], None]
     ] = None,
 ) -> torch.Tensor:
+    r"""NST from :cite:`LW2016`.
+
+    Args:
+        content_image: Content image for the NST.
+        style_image: Style image for the NST.
+        impl_params: If ``True``, uses the parameters used in the reference
+            implementation of the original authors rather than what is described in
+            the paper.
+        criterion: Optimization criterion. If omitted, the default
+            :func:`~pystiche_papers.li_wand_2016.perceptual_loss` is used. Defaults to
+            ``None``.
```

Comment:
Not going to mark this again. Please apply this everywhere.

```suggestion
        criterion: Optimization criterion. If omitted, the default
            :func:`~pystiche_papers.li_wand_2016.perceptual_loss` is used.
```